CHANGELOG.md

```diff
@@ -13,8 +13,10 @@ Version 0.1.1
 
 * Deprecate the `ServiceBaseMeta` metaclass and the `name` argument to the
   `StatsD` dependency provider.
+* Add support for pipeline when disabled
 * Test on older Nameko versions and against pre-release versions of
   dependencies.
+* Test on Python 3.8
```

Comment:
Also mention dropping support for Nameko 2.6

Reply:
Support for 2.6 was not released yet. It's in the list of changes on master but not released https://github.com/sohonetlabs/nameko-statsd/compare/0.1.0...HEAD

But I'll clarify the text to mention 2.7 as the oldest supported.

Reply:
👌